src/simplewallet/simplewallet.cpp

```diff
@@ -450,6 +455,7 @@ namespace
     catch (const tools::error::zero_destination&)
     {
       fail_msg_writer() << tr("one of destinations is zero");
+      warn_of_possible_attack = false;
```

Comment:
Aren't `tx_sum_overflow` and `tx_too_big` also exempt as locally identified errors?

Reply:
tx_sum_overflow, probably. tx_too_big, I'm not sure it can't be caused by the daemon, I erred on the side of warning.